src/cli/login.js

```diff
@@ -1,7 +1,7 @@
 import { loginUser } from '../commands/login';
 import { executeAndHandleError } from '../services/error-handler';
 
-export const description = 'Log in and register as a Shoutem developer.\nProvide credentials in username:password format.';
+export const description = 'Logs user in and registers as a Shoutem developer on first login with given credentials.\n You can provide credentials in username:password format to skip the interactive menu.';
```

Comment:
Handles user's authentication. Existing users are logged in and new users are registered as Shoutem developers. Credentials can be provided in the \`username:password\` format to skip the interactive menu.

Reply:
This isn't accurate unfortunately. Users logging in are already existing users. I agree with the second sentence.

Reply:
Handles Shoutem developer's authentication. Users already registered as Shoutem developers are simply logged in and non-developer users are first registered as developers and then logged in. Credentials can be provided in the `username:password` format to skip the interactive menu.